analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -357,17 +358,24 @@ private fun resolveConfiguration(
     analyzerResult: AnalyzerResult,
     curationProviders: List<Pair<String, PackageCurationProvider>>
 ): ResolvedConfiguration {
-    val packageCurations = mutableListOf<PackageCuration>()
+    val packageCurations = mutableMapOf<String, Set<PackageCuration>>()
 
     curationProviders.forEach { (id, curationProvider) ->
         val (curations, duration) = measureTimedValue {
-            curationProvider.getCurationsFor(analyzerResult.packages).distinct()
+            curationProvider.getCurationsFor(analyzerResult.packages)
```

Comment:
The `distinct()` should have actually been removed as part of 7ab57c2a2ca040bfd3b21542fb1b47d72c495af7. For clarity, could you please extract this to a preceding commit that is documented as a fixup for 7ab57c2a2ca040bfd3b21542fb1b47d72c495af7?